Classes/Labels/LabelsViewController.swift

```diff
@@ -10,7 +10,13 @@ import UIKit
 import IGListKit
 import Squawk
 
-final class LabelsViewController: BaseListViewController2<String>, BaseListViewController2DataSource {
+protocol LabelsViewControllerDelegate {
+    func didSelect()
+}
```

Comment:
Can this protocol be removed? Looks unused.

Reply:
Yes it is an artifact of my development, I will remove it.